ISICbalance/ViewControllers/AccountViewController.swift

```diff
@@ -104,21 +108,22 @@ final class AccountViewController: BaseViewController, ValidateErrorPresentable
     func setupBindings() {
         usernameTextField <~> viewModel.username
         passwordTextField <~> viewModel.password
-        loginButton.reactive.isEnabled <~ viewModel.loginAction.isExecuting.negate()
+        loginButton.reactive.isEnabled <~ viewModel.actions.loginAction.isExecuting.negate()
         
-        viewModel.loginAction.errors
+        viewModel.actions.loginAction.errors
             .observe(on: UIScheduler())
             .observeValues { [weak self] _ in
-                self?.presentValidationError("You must fill out all fields.")
+                self?.presentValidationError(L10n.Validate.errorMessage)
             }
 
-        viewModel.loginAction.completed.producer.startWithValues { [weak self] in
-            self?.navigationController?.popViewController(animated: true)
+        viewModel.actions.loginAction.completed.producer.startWithValues { [weak self] in
+            guard let self = self else { return }
+            self.flowDelegate?.balanceActionCompleted(in: self)
```

Comment:
můžeš klidně udělat `self?.flowDelegate?.balanceActionCompleted(in: self!)`. Je to kratší a neprůstřelný a nemusíš řešit `guard`

Reply:
ok, nechtěl jsem tam dávat !, tak jsem právě přidal guard.